src/test/java/com/jcabi/dynamo/AttributeUpdatesTest.java

```diff
@@ -229,37 +230,56 @@ public void addsMap() {
     }
 
     /**
-     * AttributesUpdates.put(...) throws exception when called.
+     * AttributesUpdates can throw exception when put is called.
      */
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void putThrowsException() {
-        new AttributeUpdates().put(
-            "key9",
-            Mockito.mock(AttributeValueUpdate.class)
-        );
+        try {
+            new AttributeUpdates().put(
+                "key9", Mockito.mock(AttributeValueUpdate.class)
+            );
+            Assert.fail("#put should not be supported");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("passed test", Matchers.notNullValue());
+        }
     }
 
     /**
-     * AttributesUpdates.putAll(...) throws exception when called.
+     * AttributesUpdates can throw exception when putAll is called.
      */
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void putAllThrowsException() {
-        new AttributeUpdates().putAll(new AttributeUpdates());
+        try {
+            new AttributeUpdates().putAll(new AttributeUpdates());
+            Assert.fail("#putAll should not be supported.");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("test ok", Matchers.notNullValue());
+        }
     }
 
     /**
-     * AttributesUpdates.remove(...) throws exception when called.
+     * AttributesUpdates can throw exception when remove is called.
      */
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void removeThrowsException() {
-        new AttributeUpdates().remove("object to remove");
+        try {
+            new AttributeUpdates().remove("object to remove");
+            Assert.fail("#remove should not be supported.");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("passed", Matchers.notNullValue());
+        }
     }
 
     /**
-     * AttributesUpdates.clear() throws exception when called.
+     * AttributesUpdates can throw exception when clear is called.
      */
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void clearThrowsException() {
-        new AttributeUpdates().clear();
+        try {
+            new AttributeUpdates().clear();
+            Assert.fail("#clear should not be supported.");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("test passed", Matchers.notNullValue());
```

Comment:
@mkordas  These always-true asserts are done since catch block cannot be empty and also a boolean flag and an if() statement afterwards cannot be done because every variable has to be final.